extensions/wikia/ArticleVideoMVP/images/close.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="10px" height="10px" viewBox="0 0 10 10" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 42 (36781) - http://www.bohemiancoding.com/sketch -->
+    <title>Close</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Test-Concepts" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" stroke-linecap="round">
+        <g id="Concept-3" transform="translate(-19.000000, -1167.000000)" stroke-width="2" stroke="#FFFFFF">
+            <g id="Video---Scroll-Copy" transform="translate(12.000000, 1160.000000)">
```

Comment:
Can we simplify those svgs and make sure we get rid of crap like comments?

https://jakearchibald.github.io/svgomg/
SVGO (including https://github.com/svg/svgo/blob/master/plugins/convertTransform.js )

BTW: did we consider using Design System icons?